prometheus.ini.example

```diff
@@ -8,3 +8,18 @@ refresh_interval = 10
 api = https://ganeti01.cluster.my-org.net:5080
 user = readonly_user
 password = secret
+
+[htools]
+# This defaults to False
+#hspace_enabled = True
+#hspace_path = /usr/bin/hspace
+# Please specifiy the disk template you would like to simulate
+#hspace_disk_template = drbd
+# Please specfy the instance specs for the simulation as $disk,$memory,$vcpus (disk/memory are expected in Megaybtes)
```

Comment:
typo: specify

Reply:
And megabytes

Reply:
wow, managed to misspell specify twice :smile: